src/components/forms/forms.js

```diff
@@ -232,6 +255,35 @@ const formInputSelect = (config) => {
   ]);
 };
 
+const formInputRadioButtons = (config) => {
```

Comment:
I also added this in mine! I haven't added tests yet but mine also has the ability to have 'Other' with text fields and such. Not sure how exactly we should merge them. My code is here https://github.com/DataBiosphere/duos-ui/pull/1747 if you want to check it out. It's a bit more complicated but it would work here too.

Reply:
these merges are going to be rough! I added some radio buttons too, but yall wrote way better code XD